bytecode.cpp

```diff
@@ -390,7 +390,10 @@ void bc_disasm(std::ostream& pyc_output, PycRef<PycCode> code, PycModule* mod,
             case Pyc::LOAD_METHOD_A:
             case Pyc::LOAD_FROM_DICT_OR_GLOBALS_A:
                 try {
-                    formatted_print(pyc_output, "%d: %s", operand, code->getName(operand)->value());
+                    auto arg = operand;
+                    if (opcode == Pyc::LOAD_ATTR_A && mod->verCompare(3, 12) >= 0 && (arg & 1))
```

Comment:
```suggestion
                    if (opcode == Pyc::LOAD_ATTR_A && mod->verCompare(3, 12) >= 0)
```
The shift in Python 3.12+ is unconditional (the flag stored at the lowest order bit is used independently of the name lookup)